terra-jupyter-base/Dockerfile

```diff
@@ -100,6 +98,7 @@ RUN pip3 -V \
  && pip3 install python-datauri \
  && pip3 install jupyter_contrib_nbextensions \
  && pip3 install jupyter_nbextensions_configurator \
+ && pip3 install markupsafe==2.0.1 \
```

Comment:
This was needed in every python image starting with the base. MarkupSafe released a new version that is not compatible with a package that `samtools` installs. We reinstall samtools or update packages in every image which was why we needed to add this line to all python images.